### PR TITLE
skip tests Gd complex workflow v2

### PR DESCRIPTION
**Pull request by gdlex4015**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by gdlex4015** [comment]
This is a brand new script to run complex workflow against V2 of papi api

File: automation/complex-prod-workflow-testV2.sh [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    do
+        checkToken $user
+    done
+
+    if [ "$NEED_TOKEN" = true ]; then
+       exit 1
+    fi
+
+
+if [ $ENV = "prod" ]; then
```
Comment (aweng98) [inline automation/complex-prod-workflow-testV2.sh:107]: check of $ENV can be done at line 14 at the beginning of script.

File: automation/complex-prod-workflow-testV2.sh [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    #submissionId=$(curl -X GET --header 'Accept: application/json' --header "Authorization: Bearer $ACCESS_TOKEN" "https://api.firecloud.org/workspaces/$namespace/$name/submissions"| jq -r '.[] | select(.status == ("Submitted")) | .submissionId')
+}
+
+monitorSubmission() {
+    user=$1
+    namespace=$2
+    name=$3
+    submissionId=$4
+
+    ACCESS_TOKEN=`docker run --rm -v $WORKING_DIR:/app/populate -w /app/populate broadinstitute/dsp-toolbox python get_bearer_token.py "${user}" "${JSON_CREDS}"`
```
Comment (aweng98) [inline automation/complex-prod-workflow-testV2.sh:90]: `ACCESS_TOKEN` cmd repeated.

**Comment by aweng98** [comment]
Are you certain about new script should be created in firecloud-ui repo. I feel this script doesn't belong here.

**Comment by gdlex4015** [comment]
The rest of the scripts live in firecloud ui. this one might as well be there too

File: automation/complex-prod-workflow-testV2.sh (hunk 1 above)
Comment (gdlex4015) [reply automation/complex-prod-workflow-testV2.sh:107]: But on line 14 I am making sure env is not empty?

File: automation/complex-prod-workflow-testV2.sh (hunk 2 above)
Comment (gdlex4015) [reply automation/complex-prod-workflow-testV2.sh:90]: Do you mean move it out of the function and to the top of the script?

File: automation/complex-prod-workflow-testV2.sh (hunk 1 above)
Comment (aweng98) [reply automation/complex-prod-workflow-testV2.sh:107]: you can perform two checks in one place.

File: automation/complex-prod-workflow-testV2.sh (hunk 2 above)
Comment (aweng98) [reply automation/complex-prod-workflow-testV2.sh:90]: yeah i think that's a better move.

**Review by aweng98 — APPROVED** [review]

**Pull request merged**